src/cascadia/TerminalApp/defaults.json

```diff
@@ -45,8 +45,7 @@
             "icon": "ms-appx:///ProfileIcons/{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.png",
             "padding": "8, 8, 8, 8",
             "snapOnInput": true,
-            "useAcrylic": true,
-            "acrylicOpacity": 0.75
+            "useAcrylic": false

```

Comment:
please retain `acrylicOpacity`, as it is illustrative when present.

Reply:
... but may be misleading when `useAcrylic` is disabled, no? Shouldn't we just specify an opacity when Acrylic is enabled?